static/sass/custom/_header.scss

```diff
@@ -4,209 +4,8 @@ $breakpoint-navigation: 980px;
 
 $color--background: #333333;
```

Comment:
Can you use the built in Vanilla variable for this? `$color-navigation-background`